RuntimeUnityEditor/Windows/ObjectTree/ObjectTreeViewer.cs

```diff
@@ -172,8 +175,18 @@ private void DisplayObjectTreeHelper(GameObject go, int indent, ref int currentC
                 }
                 GUILayout.EndHorizontal();
 
-                if (needsHeightMeasure && Event.current.type == EventType.repaint)
-                    _singleObjectTreeItemHeight = Mathf.CeilToInt(GUILayoutUtility.GetLastRect().height);
+                if (Event.current.type == EventType.repaint)
+                {
+                    if (needsHeightMeasure)
```

Comment:
This is the fix here, adding margin to the _singleObjectTreeItemHeight. Since margin is used all about I added it as a member variable as well.